### PR TITLE
Global tweaks

### PR DESCRIPTION
**Pull request by nicpottier**
- Changes wording on banner text
- Changes U-REPORTER STORY to STORY
- Hide U-Reporter stats for Global org
- Also update our translation files.

### CODE REVIEW

File: templates/public/index.haml [hunk 1]
```diff
@@ -129,81 +130,81 @@
...
+      .level-1-title-description.secondary-color
+      .card.card-padding
+        .row
           .col-md-4
             .hr-header.primary-color
-              -trans "RATE BY AGE"
-
-            .age-stats
-              -include "public/mobile_age_include.html"
+              -if org|config:"is_global"
```
Comment (norkans7) [inline templates/public/index.haml]: We  can remove this if block is no longer relevant

Comment (nicpottier) [inline templates/public/index.haml]: Which block do you mean?

Comment (norkans7) [inline templates/public/index.haml]: -if org|config:"is_global" to just show the title as "@ ureportglobal Members" down

**Comment by nicpottier** [comment]
Ok @norkans7 ?

**Comment by norkans7** [comment]
Yep

**Pull request merged**